src/back/spv/writer.rs

```diff
@@ -66,7 +78,7 @@ pub struct Writer {
     annotations: Vec<Instruction>,
     writer_flags: WriterFlags,
     void_type: Option<u32>,
-    lookup_type: FastHashMap<Word, crate::Handle<crate::Type>>,
+    lookup_type_new: FastHashMap<Word, LookupType>,
```

Comment:
is the name change accidental?

Reply:
True, I already commented on it, forgot to rename it back to `lookup_type`